tests/cases/docker/README.md

```diff
@@ -0,0 +1,13 @@
+Integrations
+============
+
+This repository contains `Dockerfile`s that describe how to build open source projects (usually those with complex build tasks) with a specific version of typescript. These are used for extended validations of a given typescript build.
+
+Contibuting
+-----------
+
+To add a new test:
+* Create a new folder with the name of the project

```

Comment:
do they each need a new folder? Simpler to have them in tests/cases/docker/

Reply:
They do, at least without copious additional configuration.